src/components/react-admin/ColorPicker.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import { SketchPicker } from 'react-color';
+
+const useStyles = makeStyles({
+  color: props => ({
+    width: '25px',
+    height: '25px',
+    backgroundColor: props.disabled ? '#fff' : props.value,
+    cursor: props.disabled ? 'default' : 'pointer',
+    backgroundImage: props.disabled
+      ? `repeating-linear-gradient(
+      45deg,
+      transparent,
+      transparent 15px,
+      rgb(255, 123, 123) 15px,
+      rgb(255, 123, 123) 20px
+        ),
+      repeating-linear-gradient(
+        135deg,
+        transparent,
+        transparent 15px,
+        rgb(255, 123, 123) 10px,
+        rgb(255, 123, 123) 20px
+      )`
+      : '',
+  }),
+});
+
+const popover = {
+  position: 'absolute',
+  zIndex: '2',
+};
+
+const cover = {
+  position: 'fixed',
+  top: '0px',
+  right: '0px',
+  bottom: '0px',
+  left: '0px',
+};
+
+const presetColors = [];
+
+const ColorPicker = ({ value = '#ccccccff', onChange, disabled }) => {
```

Comment:
a default value to `onChange` could be accurate to avoid app to crash if not filled.
Or change l.61 to `onChange?.(newColor.hex)`;

Reply:
Done